frontend/src/components/Graph/CustomGraphTooltip.tsx

```diff
@@ -11,15 +11,15 @@ interface Payload {
 
 interface Props {
   active?: boolean;
-  label?: string;
   payload?: Payload[];
 }
 
-const formatLine = (name: string, value: number) => {
-  const unit = name.split(' ')[1] === 'HR' ? ' bpm' : ' W';
-  return [`${name.split(' ')[0]}`, value, unit];
+const formatLine = (label: string, value: number) => {
+  const unit = label.split(' ')[1] === 'HR' ? ' bpm' : ' W';
+  const name = label.split(' ')[0];
+  return [`${name === 'You-Untracked' ? 'You' : name}`, value, unit];
```

Comment:
Ikke veldig viktig og kanskje bare jeg som er lett forvirra, men at man sender inn .name som label-argument og så henter ut name fra det igjen føler jeg skurrer litt.
men er jo en privat og ikke veldig viktig funksjon, så ikke noe big deal

Reply:
Yeah - optimally it would be renamed to label, but `recharts` apparently needs it to be called `name` :^)